vmm/Cargo.toml

```diff
@@ -28,7 +28,7 @@ event_monitor = { path = "../event_monitor" }
 hypervisor = { path = "../hypervisor" }
 lazy_static = "1.4.0"
 libc = "0.2.95"
-linux-loader = { version = "0.3.0", features = ["elf", "bzimage", "pe"] }
+linux-loader = { git = "https://github.com/jongwu/linux-loader", branch = "uefi", version = "0.3.0", features = ["elf", "bzimage", "pe"] }
```

Comment:
Currently we are using some source code that is on review. And the code there is a bit out-of-date. @jongwu will rebase it soon.